spriter/definitions/CharacterMap.hx

```diff
@@ -1,5 +1,5 @@
 package spriter.definitions;
-import haxe.xml.Fast;
+import haxe.xml.Access;
```

Comment:
Can you keep haxe.xml.Fast for haxe < 4? I think it can be achieved quickly using typedef.

Reply:
Ah, I didn't realize Access wasn't in Haxe 3. Now that I've read the documentation it makes me think that Access is literally just Fast renamed. I fixed it and built my changes against both 3.4.7 and 4.0.0-rc.2.